projects/fal/src/fal/auth/__init__.py

```diff
@@ -154,6 +134,24 @@ class UserAccess:
     _exc: Exception | None = field(repr=False, default=None)
     _accounts: list[dict] | None = field(repr=False, default=None)
 
+    def login(self):
+        token_data = auth0.login()
+        with local.lock_token():
+            local.save_token(token_data["refresh_token"])
+
+        self.invalidate()
```

Comment:
`invalidate` exists _because_ it was a global user. I think now that it is not global, you could

1. get rid of `invalidate` entirely
2. move `login` and `logout` outside of the class
3. make them NOT interact with the class at all